detection_rules/docs.py

```diff
@@ -24,6 +24,8 @@
 from .rule_loader import DeprecatedCollection, RuleCollection
 from .utils import load_etc_dump, save_etc_dump
 
+DOCS_DIR = "docs-dev"
```

Comment:
The entire file, is about generating docs in security-docs repository. 

Which i feel they are not using for 9.0.0, why are we introducing new directory structure here ?

the package directory being referenced here is for https://github.com/elastic/security-docs/tree/8.17/docs/detections/prebuilt-rules/downloadable-packages

rule details are here https://github.com/elastic/security-docs/tree/8.17/docs/detections/prebuilt-rules/rule-details

Reply:
thank you for spotting this! should be fixed now!